Mathlib/Order/InitialSeg.lean

```diff
@@ -194,29 +196,18 @@ protected theorem acc (f : r ≼i s) (a : α) : Acc r a ↔ Acc s (f a) :=
 
 end InitialSeg
 
-/-!
-### Principal segments
-
-Order embeddings whose range is a principal segment of `s` (i.e., an interval of the form
-`(-∞, top)` for some element `top` of `β`). The type of these embeddings from `r` to `s` is called
-`PrincipalSeg r s`, and denoted by `r ≺i s`. Principal segments are in particular initial
-segments.
--/
-
+/-! ### Principal segments -/
 
-/-- If `r` is a relation on `α` and `s` in a relation on `β`, then `f : r ≺i s` is an order
-embedding whose range is an open interval `(-∞, top)` for some element `top` of `β`. Such order
-embeddings are called principal segments -/
+/-- If `r` is a relation on `α` and `s` in a relation on `β`, then `f : r ≺i s` is an initial
+segment embedding whose range is `Set.Iio top` for some element `top`. If `β` is a well-order, this
```

Comment:
I don't like the variable name `top` here, especially because this probably won't be the top element of the order

Reply:
I'm compelled to agree, but I'm not sure what a good replacement might be.

Reply:
I think just `x` should be okay here - everything else is already a single-letter name in this context